### PR TITLE
feat: Add tests for connecting to Ethereum

### LINKED ISSUE #26: feature: Add tests for connecting to Ethereum

### PR DESCRIPTION
**Pull request by wal33d006**
Fixes #26

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -30,6 +30,10 @@ dependencies:
       ref: account-url-fix
   http: ^0.13.1
   clipboard: ^0.1.3
+  web3dart: ^2.1.3
+  wallet_core:
+    git:
+      url: https://github.com/wal33d006/wallet_core.git
```
Comment (andrzejchm) [inline pubspec.yaml:36]: I see this is a forked version of wallet_core, what is the difference between this and the original wallet_core?

File: test/etherium_test.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import 'dart:math';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart';
+import 'package:wallet_core/wallet_core.dart';
+import 'package:web3dart/web3dart.dart';
+
+void main() {
+  test('Etherium test', () async {
```
Comment (andrzejchm) [inline test/etherium_test.dart:9]: this is more of a discovery test, could we by default mark it as `skip: true` so it won't take part in CI/CD testing pipeline?

File: pubspec.yaml (hunk 1 above)
Comment (wal33d006) [reply pubspec.yaml:36]: The library was outdated, not migrated to null-safety. I migrated it and create a PR in the original repo [here](https://github.com/fuseio/wallet_core/pull/64). Till the owner migrates it to null-safety, or my PR gets approved, I am using my own forked version.

Comment (andrzejchm) [reply pubspec.yaml:36]: makes sense, thanks! definetely lets stick with the forked version for now :)

File: test/etherium_test.dart (hunk 2 above)
Comment (wal33d006) [reply test/etherium_test.dart:9]: Sounds good. Let me update that.

Comment (wal33d006) [reply test/etherium_test.dart]: Done

**Review by andrzejchm — APPROVED** [review]
great job! 🚀

File: test/etherium_test.dart [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import 'dart:math';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart';
+import 'package:wallet_core/wallet_core.dart';
+import 'package:web3dart/web3dart.dart';
+
+void main() {
+  test(
+    'Etherium test',
```
Comment (fadeev) [inline test/etherium_test.dart:10]:
```suggestion
    'Ethereum test',
```

**Review by fadeev — CHANGES_REQUESTED** [review]
Let's rename the file to `ethereum_test.dart`. Otherwise, looks good! 👍

File: test/etherium_test.dart (hunk 3 above)
Comment (wal33d006) [reply test/etherium_test.dart]: Done ✅

**Comment by wal33d006** [comment]
> Let's rename the file to `ethereum_test.dart`. Otherwise, looks good! 👍

Done ✅

**Review by fadeev — APPROVED** [review]

**Pull request merged**